cards/src/main/resources/cards/custom/doombubbles/mage/spell_meteor_storm.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Meteor Storm",
+  "baseManaCost": 8,
+  "type": "SPELL",
+  "heroClass": "BLUE",
+  "rarity": "RARE",
+  "description": "Deal 8 damage to a random minion. If it dies, cast this again.",
+  "targetSelection": "NONE",
+  "spell": {
+    "class": "custom.RecastIfMinionsDestroyedSpell",
```

Comment:
This is pretty tricky.

`RecastIfMinionsDestroyedSpell` will recast if any minion is destroyed, not the specific one that was targeted. We could conceive of a universe where there's a card on the board that says, "Whenever a minion takes damage, deal damage to this one instead," which would screw up this card. Kind of like Ramshield, but for minions.

However, it appears you can't use a normal `RecastWhileSpell` with an `IsDeadCondition`, because the condition is checked against the randomly chosen target of the `RecastWhileSpell`, not the randomly chosen target of the `DamageSpell`.

This seems fine for doing a custom spell. Carefully look at `RecastIfMinionsDestroyedSpell` to see how you can extend `RecastWhile` pretty easily to do this. However...

A programmery-way of achieving this without a custom spell class involves using an enchantment. You can create an `enchantment_meteor_storm_target` (see `enchantment_marked` for an example).

 1. `RecastWhileSpell`, whose `"spell": {"class": "MetaSpell", "spells": [...`
    1. Removes all `enchantment_meteor_storm_target` enchantments with `RemoveEnchantmentSpell`.
    2. `AddEnchantmentSpell` with `"card": "enchantment_meteor_storm_target"` to a random target
    3. Deals damage to all minions with that enchantment using `HasEnchantmentFilter`.
 2. ... and whose condition looks like:
```
    "condition": {
      "class": "AnyMatchFilterCondition",
      "target": "ALL_MINIONS",
      "filter": {
        "class": "HasEnchantmentFilter",
        "card": "enchantment_meteor_storm_target"
      },
      "invert": true
    }
```
 3. Removes all `enchantment_meteor_storm_target` enchantments with `RemoveEnchantmentSpell`.